packages/components/src/components/hds/app-header/index.hbs

```diff
@@ -25,7 +25,12 @@
     />
   {{/if}}
 
-  <div class="hds-app-header__actions" id={{this.menuContentId}}>
+  {{! if it's not desktop view and the menu is not open, we want to hide the actions from assistive tech }}
+  <div
+    class="hds-app-header__actions"
+    id={{this.menuContentId}}
+    aria-hidden="{{(and (not this.isDesktop) (not this.isOpen))}}"
```

Comment:
question: do we _need_ `aria-hidden` to be set? `display: none` should be enough to remove these items from the accessibility tree

Reply:
Thanks, I will update to remove.

Reply:
Removed